neptune/new/internal/init/run.py

```diff
@@ -99,102 +99,107 @@ def init_run(
-        capture_stderr (bool, optional):  Whether to send run’s stderr. Defaults to `True`.
-            Tracked metadata will be stored inside `monitoring_namespace`.
-        capture_hardware_metrics (bool, optional): Whether to send hardware monitoring logs
-            (CPU, GPU, Memory utilization). Defaults to `True`.
-            Tracked metadata will be stored inside `monitoring_namespace`.
-        fail_on_exception (bool, optional): Whether to register an uncaught exception handler to this process and,
-            in case of an exception, set run's sys/failed to True. Exception is always logged
-        monitoring_namespace (str, optional): Namespace inside which all monitoring logs be stored.
+        project: Name of the project where the run should go, in the form "workspace-name/project_name".
+        If None (default), the value of the NEPTUNE_PROJECT environment variable is used.
+        api_token: User's API token. Defaults to None.
+            If None (default), the value of the NEPTUNE_API_TOKEN environment variable is used.
+            Note: To keep your API token secure, save it to the NEPTUNE_API_TOKEN environment variable rather than
+            placing it in plain text in the source code.
+        with_id: If you want to resume a run, the identifier of the existing run.
+            For example, 'SAN-1'. A run with such an ID must exist.
+            If None (default) is passed, starts a new tracked run.
+        custom_run_id: A unique identifier to be used when running Neptune in pipelines.
+            Make sure to use the same identifier throughout the whole pipeline execution.
+        mode: Connection mode in which the tracking will work.
+            If None (default), the value of the NEPTUNE_MODE environment variable is used.
+            If no value was set for the environment variable, 'async' is used by default.
+            Possible values are 'async', 'sync', 'offline', 'read-only', and 'debug'.
+        name: Editable name of the run. Defaults to 'Untitled'.
+            The name is displayed in the run details and as a column in the runs table.
+        description: Editable description of the run. Defaults to `''`.
+            The description is displayed in the run details and can be added to the runs table as a column.
+        tags: Tags of the run as a list of strings. Defaults to `[]`.
+            Tags are displayed in the run details and in the runs table as a column.
+            You can edit the tags after the run is created, either through the app or the API.
+        source_files: List of source files to be uploaded.
+            Uploaded source files are displayed in the 'Source code' tab of the run view.
+            Unix style pathname pattern expansion is supported. For example, you can pass `*.py` to upload
+            all Python files from the current directory.
+            If None is passed, the Python file from which the run was created will be uploaded.
```

Comment:
People often ask about turning this off and solution is empty list [], maybe we can add it to the description?

Reply:
Good idea